README.md

```diff
@@ -15,22 +15,30 @@ This can be used alongside other plugins such as [@hicksy/arc-plugin-sandbox-str
 
 1. Install this package using npm:
 
-       npm install -D @nasa-gcn/architect-plugin-dynamodb-local
+```
+npm install -D @nasa-gcn/architect-plugin-dynamodb-local
+```
 
 1. In your `.env` add the following:
 
-       ARC_TABLES_PORT=8000
-       ARC_DB_EXTERNAL=true
+```
+ARC_TABLES_PORT=8000
+ARC_DB_EXTERNAL=true
```

Comment:
Will this work correctly with the `external-db` setting in the [`@sandbox` setting of the local preferences](https://arc.codes/docs/en/reference/cli/sandbox#%40sandbox)? My concern is that we exclude `.env` from version control, and so this would be an extra manual configuration step.

Reply:
Actually this is a pretty significant update. This will have override some of the work that the sandbox was already doing. I will keep working on this, but track it as a new feature